x-pack/platform/packages/shared/kbn-grok-heuristics/index.ts

```diff
@@ -7,7 +7,7 @@
 
 export { getReviewFields } from './src/review/get_review_fields';
 export { getGrokPattern } from './src/review/get_grok_pattern';
-export { unwrapPatternDefinitions } from './src/review/unwrap_pattern_definitions';
+export { unwrapPatternDefinitions } from '@kbn/streamlang/src/transpilers/esql/processors/grok';
```

Comment:
@thomheymann with streamlang supporting pattern definitions, should we even unwrap them for generated grok patterns, or keep them separate?